tests/test_json_schema.py

```diff
@@ -287,6 +288,20 @@
                 'type': 'object',
             },
         ),
+        (
+            ObjectIdField,
+            {
+                'properties': {
+                    'x': {
+                        'title': 'X',
+                        'type': 'string',
+                    }
+                },
+                'required': ['x'],
+                'title': 'Model',
+                'type': 'object',
+            },
+        ),
```

Comment:
```suggestion
        (
            ObjectIdField,
            {
                'properties': {
                    'x': {
                        'anyOf': [{'type': 'string'}],
                        'title': 'X',
                    }
                },
                'required': ['x'],
                'title': 'Model',
                'type': 'object',
            },
        ),
```

This will fix the failed tests